common/src/main/scala/common/util/StringUtil.scala

```diff
@@ -0,0 +1,15 @@
+package common.util
+
+import mouse.all._
+import org.apache.commons.lang3.StringUtils
+
+object StringUtil {
+  implicit class EnhancedString(val string: String) extends AnyVal {
+
+    def ensureSlashed: String = StringUtils.appendIfMissing(string, "/")
+
+    def ensureUnslashed: String = string.ensureSlashed |> StringUtils.chop
+
+    def removePrefix(prefix: String): String = StringUtils.removeStart(string, prefix)
```

Comment:
Is this different from http://www.scala-lang.org/api/2.12.4/scala/collection/immutable/StringOps.html#stripPrefix(prefix:String):String ?

Reply:
My version *might* be null safe depending on how implicit classes are implemented, but the way we're using this code we'd have some pretty big problems if that argument was null. I'll remove this and use the library version.